### PR TITLE
Fix dashboard tests

### PR DESCRIPTION
**Pull request by nicele08**
# What does this PR do?
>- Create dashboard layout

# Description of Task to be completed?
>- Add sidebar navigation
>- Add navbar with search, notification, and profile popup

## Requirements

- [x] Require dependencies installation

# How should this be manually tested?
> - Clone this repository
> - checkout to this branch `ft-dashboard-layout`
> - install the dependencies yarn install` and 
> - Start app `yarn dev` 
> - Go to `http://localhost:$PORT/dashboard` in your browser to access the page

# Any background context you want to provide?
>- N/A
# What are the relevant pivotal tracker stories?
>- N/A
# Screenshots (if appropriate)
-> dashboard

# Questions:
>- N/A

### CODE REVIEW

**Review by istafrica-dennw — APPROVED** [review]

**Comment by Cherubln** [comment]
Well done @nicele08. Just a few suggestions left.

File: src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    href: '/dashboard/reports',
+    icon: HiDocumentReport,
+  },
+];
+
+const AppSidebar = ({ className = 'w-fit' }) => {
+  return (
+    <div className={className}>
+      <Sidebar aria-label="Sidebar with logo branding example">
+        <Sidebar.Logo href="#" img={logo} imgAlt="Flowbite logo">
```
Comment (cherubln) [inline src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx]: The logo image well seen on dark theme, lets put original logo as this one is not clearly seen on light theme.

File: src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          CodeHills
```
Comment (cherubln) [inline src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx:16]: Can we remove this codehills text on small devices and place a search button in left which will open a search input on click?

File: src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                </Sidebar.Item>
+              </NavLink>
+            ))}
+          </Sidebar.ItemGroup>
+
+          <Sidebar.ItemGroup>
+            <div className="relative flex items-center w-full">
+              <DarkThemeToggle about="Dark mode" title="Dark Mode" />
+              <p className="left-10 text-gray-500 dark:text-gray-400 absolute pointer-events-none">
+                Dark mode
```
Comment (cherubln) [inline src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx:60]: For better UX can we change the text(light/dark) accordingly as you have done on the icons?

File: src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  HiClock,
+  HiDocumentReport,
+  HiChartBar,
+  HiCheckCircle,
+} from 'react-icons/hi';
+import { NavLink } from 'react-router-dom';
+
+import logo from '@/assets/images/logos/logo.svg';
+
+const navLinks = [
```
Comment (cherubln) [inline src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx]: What if we put those navLinks in a separate folder like **Constants** for future scalability and component's lines reduction?

**Review by Cherubln — CHANGES_REQUESTED** [review]

File: src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx (hunk 1 above)
Comment (nicele08) [reply src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx]: I see

**Review by Cherubln — APPROVED** [review]
LTM

**Comment by Cherubln** [comment]
>  Can we remove this codehills text on small devices and place a search button in left which will open a search input on click? 

I think you have decided to change the whole search input into a search button, But I was actually supposing this to be done only on small devices. But it is fine we can work on that as we go.

**Review by istafrica-dennw — APPROVED** [review]

**Pull request merged**